ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -298,32 +298,47 @@ function wait_clusteroperators_continous_success() {
 }
 
 function check_mcp() {
-    local updating_mcp unhealthy_mcp
-
-    updating_mcp=$(oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status --no-headers | grep -v "False")
-    if [[ -n "${updating_mcp}" ]]; then
-        echo "Some mcp is updating..."
-        echo "${updating_mcp}"
+    local updating_mcp unhealthy_mcp tmp_output
+
+    tmp_output=$(mktemp)
+    oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status --no-headers > "${tmp_output}"
```

Comment:
the script will exit here when the command fail?

Reply:
Ah, good catch, later I will append `|| true` to ensure it works.

Reply:
fixed.